lib/ScinanDevice.js

```diff
@@ -193,10 +205,11 @@ class ScinanDevice extends Homey.Device {
     // await this.updateName();
   }
 
-  async onSettings(oldSettings, newSettings) {
+  async onSettings({oldSettings, newSettings, changedKeys}) {
     const { interval } = this;
```

Comment:
Denne er vel ubrukt nå, så kan slettes

Reply:
Nei. onSettings brukes fortsatt for å oppdatere Interval i objektet.